Expression Evaluator

cmsbuild opened the pull request:
A new Expression Evaluator based on the standard C++ compiler
to replace the current one that is implemented using on a home-made parser (based on boost::spirit) and access objects using ROOT-reflection

motivations, doc, examples in 
http://innocent.home.cern.ch/innocent/ExprEval.pdf
presented at  reco/phys-tool meeting on 29/01
Automatically ported from CMSSW_7_4_X #7480

smorovic committed:
output TRANSFERS_PSET.jsn from the EvFDaqDirector service

smorovic committed:
*writing transfers pset file for all streams produced by CMSSW

smorovic committed:
std::string --> std::vstring (to have list of destinations per each
stream and mode)

smorovic committed:
creation of BoLS file

smorovic committed:
transfer destination added to output json for all streams (except streamError which is external)

smorovic committed:
fake BU support for copying existing directory with HLT menu and other parameters propagated through BU

smorovic committed:
ini file checksum: output module reads back and verifies checksum of the
ini file. then file is moved from open subdirectory to the local run
directory

smorovic committed:
sleep and repeat check for EoL two times in case EoR appears without EoL for current lumisection

smorovic committed:
nFeds can now be beyond 1024 in fake BU output module

cmsbuild commented:
A new Pull Request was created by @cmsbuild  for CMSSW_7_5_X.

Expression Evaluator

It involves the following packages:

CommonTools/CandUtils
CommonTools/RecoUtils
CommonTools/Utils


@nclopezo, @cvuosalo, @monttj, @cmsbuild, @slava77, @vadler can you please review it and eventually sign? Thanks.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@nclopezo, @ktf you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

nclopezo commented:
+1
Tests already approved in the equivalent of this PR in CMSSW_7_4_X

makortel committed:
Remove const_cast

makortel committed:
Migrate bunch of producers to stream::EDProducer

makortel committed:
Migrate bunch of producers to global::EDProducer

makortel committed:
Migrate TrackerSeedValidator to DQMEDAnalyzer

It looks like this was partially done earlier, but the change of base
class was missing.

makortel committed:
Add overrides to MultiTrackValidator

makortel committed:
Clean ParameterSet members from CosmicTrackFinder and CosmicSeedGenerator

matteosan1 committed:
add fillDescriptions to EcalUncalibRechitProducer

matteosan1 committed:
cleaned producer

Dr15Jones committed:
Fix typo in DQMHelper

The DQMHelper was using xmax instead of xmin so that the histograms
had no range. This lead to failures when the DQMStore tried to merge
histograms while running in the threaded framework.

Dr15Jones committed:
add comments

matteosan1 committed:
use ifValue

slava77 commented:
@VinInn 
I wasn't sure if this PR still expecting some developments from you to reduce the size of the precompiled headers.
Please clarify.

matteosan1 committed:
added default config

VinInn commented:
My ideas was to release it as is to see the effect and start porting
@davidlange6 , what is your opinion?

matteosan1 committed:
final version

ptcox committed:
fix misnamed header guard

matteosan1 committed:
fix in pset name

matteosan1 committed:
fix in pset name

matteosan1 committed:
fix of default cfi

matteosan1 committed:
fix of default cfi

ptcox committed:
remove const_cast that looks unnecessary to me anyway

matteosan1 committed:
finally fixed multifit config

matteosan1 committed:
finally fixed multifit config

boudoul committed:
for workflow 2017

matt-komm committed:
deleted pixel/strip fastclusters

ptcox committed:
remove unnecessary static flag

matt-komm committed:
apply cmssw lib/plugin separation

ianna committed:
Get vector names

cmsbuild committed:
Merge pull request #7861 from rovere/TrackingMaterialForRecoGeometry

Tracking material for reco geometry

cmsbuild committed:
Merge pull request #7856 from dildick/for-75X-GEM-digi-custom-bugfix

GEM digi custom bugfix

cmsbuild committed:
Merge pull request #7852 from mmarionncern/74X-METSigOutputFix

Fix output format of MET covariance matrix in MET significance code

gartung committed:
update to getParamDumper and change ClassChecker ClassDumper and FunctionDumper to print filename in (anonymous namespace)

davidlange6 committed:
Merge pull request #7779 from ianna/2023-scenario

2023 Scenario Geometry

davidlange6 committed:
Merge pull request #7787 from cms-btv-pog/CSVv2AVR_from-a332248d6e1f9efe5cd45436fca02bcfd36ba97a

AVR-based CSVv2 taggers, updated SV TagInfos in MiniAOD

wmtan committed:
Fix for copnsumes interface

wmtan committed:
Add consumes interface to SimCalorimetry/EcalTrigPrimProducers

cmsbuild committed:
Merge pull request #7879 from ianna/dd-vector-names-by-string

Get DDVector Names

slava77 commented:
+1

for https://github.com/cms-sw/cmssw/pull/7807	2ba7e0c

new feature.
This partially becomes a test for the distribution infrastructure.

wmtan committed:
Refactor Wrapper.h

wmtan committed:
Add consumes interface for CSCTFTrackProducer

wmtan committed:
Add consumes interface for GctDigiToRaw

apfeiffer1 committed:
added default ctors to embedded classes to avoid unit test crashing in ROOT6 branch

cmsbuild committed:
Merge pull request #7890 from wmtan/ReorganizeWrapperHeader

Refactor  and split up Wrapper.h for maintainability, as it has become unwieldy

cmsbuild committed:
Merge pull request #7876 from ptcox/csc_geom_kill_const_cast

csc geom kill const cast

davidlange6 committed:
Merge pull request #7764 from gzevi/pat_photon_isolation_conflict_with_reco

renamed functions to access PAT isolation variables to avoid conflict wi...

cmsbuild committed:
Merge pull request #7864 from ptcox/csc_seg_fit_factored_and_smatrixed

CSC segment fit factored and smatrixed

davidlange6 committed:
Merge pull request #7886 from wmtan/ConsumesSimCalorimetryEcalTrigPrimProducers

Add consumes interface to SimCalorimetry/EcalTrigPrimProducers

davidlange6 committed:
Merge pull request #7885 from wmtan/ConsumesSimGeneralMixingModule

Fix for consumes interface in SimGeneral/MixingModule

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

davidlange6 committed:
Merge pull request #7891 from wmtan/ConsumesEventFilterGctRawToDigi

Add consumes interface for GctDigiToRaw

davidlange6 committed:
Merge pull request #7889 from wmtan/ConsumesL1TriggerCSCTrackFinder

Add consumes interface for CSCTFTrackProducer

VinInn committed:
factorize out choice of plane, use it for loopers as well

cmsbuild committed:
Merge pull request #7878 from ptcox/csc_remove_static_variable

remove unnecessary static flag

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

matt-komm committed:
deleted obsolete cfi/cff files

silviodonato committed:
HLTBTagHarvestingAnalyzer.cc

mmusich committed:
fixing Magnetic field map for track refit

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

matteosan1 committed:
Merged refs/pull/7877/head from repository cms-sw

matt-komm committed:
added notes concerning tuning/validation

matt-komm committed:
fix famos sequence

matteosan1 committed:
bugfix for ConversionTools

gartung committed:
update regex for new EventSetupRecord::get<> pattern in class-composition.py

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

Commit:
Added Dxy plots (only for electron/muon at this moment)

cmsbuild committed:
Merge pull request #7873 from deguio/fixRunIsComplete_74

fix runIsComplete behaviour for express processing

civanch committed:
added dump of regions/cuts; updated dum of the geometry

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

Martin-Grunewald committed:
Update for Ecal/fillDescription/MultiFit

makortel committed:
Remove unnecessary assert

cmsbuild committed:
Merge pull request #7912 from matt-komm/cleanupFastSimClusters

Cleanup fast sim clusters

cmsbuild committed:
Merge pull request #7909 from civanch/updated_geom_dump

Updated dump of SIM geometry

cmsbuild committed:
Merge pull request #7900 from mmusich/fixHIPmagfield

Fixing Magnetic field for track refit in HIP

cmsbuild committed:
Merge pull request #7899 from silviodonato/hltbtagval-75X

Removing exceptions in BTag HLT validation (#7881) - ported in 75X

cmsbuild committed:
Merge pull request #7800 from olschewski/dtcalib-t0febpathcorrection_74X

Added DTCalibration FEB Path Correction

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge pull request #7857 from cirkovic/from-CMSSW_7_5_X_2015-02-19-0200_FIX

Bug solution for pull request #6731

Commit:
Update Vx3DHLTAnalyzer.cc

Same modification as for 7_4_X

Commit:
Update beampixel_dqm_sourceclient-live_cfg.py

Same modification as for 7_4_X

ebrondol committed:
changed name of collection

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

Commit:
Set antiparticle decay table.  Remove unnecessary endl from Log messages.

Commit:
Add comment about spin

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

wmtan committed:
File information in playback info

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

Martin-Grunewald committed:
Merged refs/pull/7911/head from repository cms-sw

cmsbuild committed:
Merge pull request #7936 from wulsin/cleanupCustom

Cleanup custom

cmsbuild committed:
Merge pull request #7932 from dinardo/patch-6

Update beampixel_dqm_sourceclient-live_cfg.py

cmsbuild committed:
Merge pull request #7883 from gartung/clangSA-getParamDumper-update2

update to getParamDumper and change ClassChecker ClassDumper and Functio...

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge pull request #7930 from dinardo/patch-4

Update Vx3DHLTAnalyzer.cc

ianna committed:
Add new Tracker parameter record to the tests and writers. Cleanup configurations.

makortel committed:
Clean non-const statics

davidlange6 committed:
Merge pull request #7812 from bendavid/genstatusflags_740p7

Add GenStatusFlags

smorovic committed:
*moving new fffparameters.jsn file
*updating to latest json metafile definitions as in BU

makortel committed:
Fix typos and initialize pointers in SiStripTrackingRecHitsValid

makortel committed:
Make memory leak less probably in DAFTrackProducerAlgorithm

smorovic committed:
*passing selected transfer mode as untracked parameter
*updated cfi definition of EvFDaqDirector
*fixed stream destinations call in DQMFileSaver

wmtan committed:
consumes interface for L1Comparator

Commit:
plot bx for all triggers

ferencek committed:
moved soft lepton TagInfos and taggers under the candidate-based fwk

ferencek committed:
switching to the old parameter name for the PV collection

ferencek committed:
added MiniAOD support for soft lepton taggers

cmsbuild committed:
Merge pull request #7957 from wmtan/ConsumesL1Comparator

consumes interface for L1Comparator

cmsbuild committed:
Merge pull request #7917 from matteosan1/topic_conversiontools

bugfix for ConversionTools

cmsbuild committed:
Merge pull request #7916 from cms-tsg-storm/branch50nsHltMenu75X

HLT update including Ecal for 75X

cmsbuild committed:
Merge pull request #7896 from VinInn/Loopers

factorize out choice of plane, use it for loopers as well

makortel committed:
std::isinf -> edm::IsNotFinite in TwoTrackMinimumDistanceHelixHelix

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

smorovic committed:
fixed iSummary variable name (not iSummary_)

davidlange6 committed:
message logger already has a \n so no need for a second one

smorovic committed:
*DQMFileSaver fixes:
 - beginRun is const, so we must read transfer system string in beginJob
   (also could be done from constructor)
 - fillJson is static, so we pass transfer system string as parameter. Also adapted TimeoutPoolOutputModule to this.

ebrondol committed:
add TrajTrackAssociationCollection Handle

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

makortel committed:
Simplify bit operations

The results should stay the same.

ebrondol committed:
add TrajTrackMap to propagate algo and quality info, Traj vector not needed anymore

makortel committed:
Fix setting of IS_ASSOC2{FIRST,ANY}_RECO bits

cmsbuild committed:
Merge pull request #7963 from davidlange6/remove_emptylines

memory service fix - the message logger already has a \n so no need for a second one

ebrondol committed:
cleaning up

ebrondol committed:
conf test file modified

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

venturia committed:
Reorganization of the occupancy map macros

venturia committed:
use the correct GeometryRecoDB_cff configuration fragment

smorovic committed:
*can not get process Pset in constructor of the service. Subsequently also can
not get anything from constructor of the output module. beginJob is
at the moment safe to override because StreamerOutputModuleBase does not do anything
at that time, so moving the call there.

venturia committed:
Update and rename DPGAnalysis/SiStripTools to DPGAnalysis/SiStripTools/README.md

ferencek committed:
fixed inverted conversion veto condition

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

wmtan committed:
Complete the consumes interface

wmtan committed:
Add consumes interface

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

wmtan committed:
Add consumes interface

makortel committed:
Fix HLTpostProcessorVertex_cfi

The DQM directory of the original postProcessorVertex was modified.
This bug didn't affect the central workflows, as both
postProcessorVertex (processing HLT folder) and
postProcessorHLTvertexing (processing the reco folder) were run. But
running the postProcessorVertexStandAlone sequence standalone was
affected.

ebrondol committed:
fix to run in 75X

ebrondol committed:
fix to run in 75X (2)

smorovic committed:
*removed unused functions constructing the transfer file path
*fake BU writing EoLS file after interrupted by signal
*supervisor thread makes a check for additional files when run is ended, delayed by 0.1 seconds.
  This makes sure that new raw data is not present. Also this slightly postpones a check
  for EoLS file in ramdisk in case it does not become visible before EoR file (when their timestamps are very close)

cmsbuild committed:
Merge pull request #7971 from wmtan/ConsumesSimHitTPAssociationProducer

Add consumes interface for  SimHitTPAssociationProducer.

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

makortel committed:
Add resolution and pull validation plots for primary vertex

Commit:
The roll script is modified in order to create the DQM global lock file.

wmtan committed:
Migrate from getByLabel to getByToken

wmtan committed:
Migrate from getByLabel to getByToken

wmtan committed:
Migrate from getByLabel to getByToken

wmtan committed:
Migrate CSCTFTrackProducer to getByToken

makortel committed:
Add resolution and pull plots separately for merged vertices

cmsbuild committed:
Merge pull request #7987 from batinkov/CMSSW_7_5_X

The roll script is modified in order to create the DQM global lock file.

cmsbuild committed:
Merge pull request #7982 from lveldere/fastsim-fix-pucontent

extend fastsim pu event content to match with latest developments in pu ...

davidlange6 committed:
Merge pull request #7814 from wmtan/NewPlaybackInfo

File information in playback info

cmsbuild committed:
Merge pull request #7908 from ndaci/ExoValDev_75X_PR1

Added Dxy plots (only for electron/muon at this moment)

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge pull request #7975 from wmtan/ConsumesObjectViewCleaner

Add consumes interface for ObjectViewCleaner

slava77devel committed:
constexpr to not need static locals

slava77devel committed:
default initialization

slava77devel committed:
initialize arrays (issues found by static analyzer)

slava77devel committed:
initialize arrays (issues found by static analyzer)

slava77devel committed:
stream::EDProducer<>

slava77devel committed:
static const (for the static analyzer)

slava77devel committed:
remove a bogus const_cast

davidlange6 committed:
Merge pull request #7969 from rappoccio/JMEStreamMigration

Switching to stream producers for JME Packages

cmsbuild committed:
Merge pull request #8001 from threus/pixclusterfix-74x

fixes of track collection and vertex selection

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

slava77devel committed:
added a default constructor

slava77devel committed:
provide an explicit initialization

davidlange6 committed:
Merge pull request #8004 from slava77/CMSSW_7_4_0_pre8/mt-safe-devA

mt safety updates around RecoLocalCalo RecoMET and JetMETCorrections/Type1MET

cmsbuild committed:
Merge pull request #7998 from diguida/fixRecoParticleFlow

Fix RecoParticleFlow producers as reported by static analyser

davidlange6 committed:
Merge pull request #7997 from wmtan/GetByTokenCSCTFTrackProducer

Migrate CSCTFTrackProducer to getByToken

davidlange6 committed:
Merge pull request #7996 from wmtan/ConsumesSimHitTPAssociationProducer

Migrate from getByLabel to getByToken for SimHitTPAssociationProducer

perrotta committed:
HIon menu and fillDescription in EcalUncalibRechitProducer

bendavid committed:
fix identification of direct tau decay products in case intermediate W appears in tau decay chain

bendavid committed:
deal properly with quarks having multiple mothers in the parton shower (for identification of hard process quarks such as b's from top or higgs decays)

cmsbuild committed:
Merge pull request #7981 from ebrondol/ChangedTracksCollections720pre4

DAF update

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

cmsbuild committed:
Merge pull request #8002 from smorovic/daqupdates+transfersPSet-75X

transferSystem PSet support and other DAQ updates (75X)

Commit:
clean up some useless headers

deguio committed:
merging test for DQM+validation workflow

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

davidlange6 committed:
Merge pull request #8006 from bendavid/genstatusflagfixes

Some fixes for GenParticle status flags

wmtan committed:
Use getByToken, not getByLabel

wmtan committed:
Use getByToken, not getByLabel

cmsbuild committed:
Merge pull request #8005 from cms-tsg-storm/continueHltIntegration75X

HIon menu and fillDescription in EcalUncalibRechitProducer

cmsbuild committed:
Merge pull request #7968 from wmtan/ConsumesL1TriggerDTTrackFinder

Complete the consumes interface for package L1Trigger/DTTrackFinder

wmtan committed:
Use getByToken, not getByLabel

cmsbuild committed:
Merge pull request #8018 from wmtan/ConsumesGctDigiToRaw

Use getByToken, not getByLabel for GctDigiToRaw

cmsbuild committed:
Merge pull request #8014 from wmtan/ConsumesSimEcalTrigPrimProducer

Use getByToken, not getByLabel, in EcalTrigPrimProducer

Commit:
Adapted to new MC flags

deguio committed:
fix naming

thomreis committed:
use isPromptFinalState() instead of genParticle status

cmsbuild committed:
Merge pull request #8030 from ndaci/ExoValDev_75X_PR2

Adapted to new MC flags

cmsbuild committed:
Merge pull request #8009 from makortel/vertexValidation

Add resolution and pull plots to vertex validation, and fix two bugs

cmsbuild committed:
Merge pull request #7958 from jingyucms/jingyu-75x-bx-plot

plot bx for all triggers

jgomez2 committed:
New HI plots

jgomez2 committed:
New HI plots

jgomez2 committed:
New HI plots

ptcox committed:
cout replaced by LogVerbatim

wmtan committed:
add consumes interface

davidlange6 committed:
Merge pull request #8024 from wmtan/ConsumesL1Comparator

Use getByToken, not getByLabel, in L1Comparator

cmsbuild committed:
Merge pull request #8041 from ptcox/cscdigis_cout_to_messagelogger

cout replaced by LogVerbatim

cmsbuild committed:
Merge pull request #8036 from thomreis/Egamma_HLT_relval_genStatusFix

use isPromptFinalState() instead of genParticle status for egamma HLT relval

davidlange6 committed:
Merge pull request #7964 from venturia/dpganalysis_sistriptools-75x

Re-organization of occupancy macros and documentation, change of the Geometry cff

cmsbuild committed:
Merge pull request #8031 from boudoul/forAndrea_weFeb20

step further for workflow 2017 (phase1)

Commit:
First try to parse millepede.end - not yet successful but had to leave

Commit:
Added check of millepede.end to mps_check.pl. Jobs will go to WARN if return code is 1, OK if 0 and FAIL otherwise.
Key to code is on the Millepede-II website: http://www.desy.de/~kleinwrt/MP2/doc/html/exit_code_page.html

cmsbuild committed:
Merge pull request #8044 from wmtan/ConsumesCastorMonitorModule

add consumes interface to CastorMonitorModule

davidlange6 committed:
Merge pull request #8011 from deguio/testMergeStep

Test merge step

cmsbuild committed:
Merge pull request #8042 from deguio/stdProcessingInMatrix

define standard DQM and Valid WF in the matrix

cmsbuild committed:
Merge pull request #8040 from jgomez2/HITrackingDQMUpdate

Hi tracking dqm update

cmsbuild committed:
Merge pull request #7892 from apfeiffer1/L1TO-CaloParams-ctors

added default ctors to embedded classes to avoid unit test crashing

deguio committed:
working in production by default

patch to be applied in case of playback deployement

patch to be applied in case of user area deployement

changing the default to users gui

adding patch for prod installation

davidlange6 committed:
Merge pull request #8020 from cms-btv-pog/PATBTaggingUpdates_from-CMSSW_7_4_0_pre7

Soft lepton tagger updates (75X)

cmsbuild committed:
Merge pull request #7983 from makortel/staticanalyzer_fix_trk

Fix issues reported by static analyzer in tracking related code

cmsbuild committed:
Merge pull request #7949 from ianna/geometry-tests-update

Add Tracker Parameters to Tests and Writers

davidlange6 committed:
Merge pull request #8045 from davidlange6/printouts_r2

Printouts r2

cmsbuild committed:
Merge pull request #8038 from deguio/fixSoftReset_74

extend the softReset use case when histos have labels on the axis

cmsbuild committed:
Merge pull request #8016 from fabozzi/from-CMSSW_7_4_X_2015-03-01-0200_updatesFor74X

updated GEN-SIM input; fixed HI wf's INPUT

perrotta committed:
Update the HLT menu for EGamma and Muon, and add a few more paths to it

rappoccio committed:
Stream migration for JME HLT modules

deguio committed:
fix

davidlange6 committed:
Merge pull request #8072 from deguio/patchesForOnline_75

Patches for online 75

cmsbuild committed:
Merge pull request #8065 from mengleisun/ECALDQM

fix ECAL summary map

cmsbuild committed:
Merge pull request #8050 from frmeier/patch_mpendparse

Parse exit state of alignment in millepede.end and set job status accordingly

cmsbuild committed:
Merge pull request #8070 from cms-tsg-storm/hltMenuUpdates_75x

Update the HLT menu for EGamma and Muon, and add a few more paths to it

wmtan committed:
complete consumes interface

ptcox committed:
remove long-superseded test

ptcox committed:
suggested fix by chris jones for a multithreading incompatibility

ptcox committed:
add bool flag to control cscdigitizer debug output

ptcox committed:
config to show use of cscdigidump

ptcox committed:
update cscdigitizer test config to 7xx

ptcox committed:
update simple cscgascollisions test to 7xx

ptcox committed:
replace cout by logverbatim

ptcox committed:
revert to eras version

ptcox committed:
add cscdigitizer dump flag

deguio committed:
remove couts

covarell committed:
removed status2

ktf committed:
Merge pull request #8091 from deguio/removeCoutsHarvesting

remove couts

cmsbuild committed:
Merge pull request #8093 from covarell/75X_exampmacro

removed status 2 from example macro (does not work in Pythia8)

cmsbuild committed:
Merge pull request #8088 from ptcox/cscdigitizer_cout_to_messagelogger

CSCDigitizer cout to MessageLogger and updates

cmsbuild committed:
Merge pull request #8081 from wmtan/ConsumesHLTRechitInRegionsProducer

complete consumes interface for /HLTRechitInRegionsProducer

cmsbuild committed:
Merge pull request #8075 from rappoccio/JMEHLTStreamMigration

Stream migration for JME HLT modules

cmsbuild committed:
Merge pull request #7798 from makortel/phase1_fwd_variousSimple

Forward-port three simple tracking PRs from 620_SLHC

cmsbuild committed:
Merge CMSSW_7_4_X into CMSSW_7_5_X.

VinInn commented:
what is the status of this?

slava77 commented:
merge conflicts are here now
Vincenzo, please check.

VinInn committed:
fix merge conflict in BUildFile

VinInn commented:
rebase failed!

cmsbuild commented:
Pull request #7807 was updated. @diguida, @lveldere, @ianna, @cerminar, @Martin-Grunewald, @covarell, @bendavid, @perrotta, @civanch, @franzoni, @cmsbuild, @fwyzard, @ktf, @davidlange6, @vciulli, @Dr15Jones, @cvuosalo, @mdhildreth, @deguio, @slava77, @ggovi, @vadler, @mmusich, @mulhearn, @ssekmen, @nclopezo, @danduggan, @boudoul, @thuer, @monttj, @srimanob can you please check and sign again.

VinInn commented:
moved to #8134

VinInn commented:
PLEASE CLOSE!
